kubernetes/samples/charts/voyager/README.md

```diff
@@ -21,7 +21,7 @@ $ helm search repo appscode/voyager
 NAME               CHART VERSION APP VERSION	DESCRIPTION
 appscode/voyager   v12.0.0       v12.0.0    	Voyager by AppsCode - Secure HAProxy Ingress Co...
 ```
-> **NOTE**: After updating the helm repository, the Voyager version listed maybe newer that the one appearing here, please check with the Voyager site for the lastest supported versions.
+> **NOTE**: After updating the helm repository, the Voyager version listed maybe newer that the one appearing here, please check with the Voyager site for the latest supported versions.
```

Comment:
helm  -> Helm (capitalize)
maybe  -> may be (two words)
here, please -> here. Please (two separate sentences)

Reply:
done